scripts/motor-expert-screen

```diff
@@ -31,10 +31,21 @@ export EDMDATAFILES=.:${PCDS_EDMS}/xps8:/reg/g/pcds/package/epics/3.14/ioc/commo
 source /reg/g/pcds/setup/pathmunge.sh
 ldpathmunge /usr/local/lib
 
+# Check to see if this is a Beckhoff axis
+#   caget will return exit code 0 if :PLC:nErrorId_RBV exists, 
+#   we set isbeckhoff based on this.
+isbeckhoff=false
+caget "${PREFIX}:PLC:nErrorId_RBV" > /dev/null 2>&1
```

Comment:
This is an extra 1 second timeout on all non-beckhoff axes. Is it possible to do these checks inside the rtyp == motor branch so that there is minimal impact on other kinds of motors?

Reply:
Yeah, sure. I originally put it outside to keep the nesting as shallow as possible but the one second timeout is not great. I will update.